source/includes/administration/_additional_roles.md

```diff
@@ -0,0 +1,115 @@
+## Additional roles
+
+With additional roles, you can allow users access to other organizations. You can specify a scope which targets one or many organizations with a specific role. See [roles](#administration-roles) for more information on roles.
+
+### Scoping
+
+There are different types of scope that you can use depending on your use case. This scope must be provided when assigning an additional role to a user. You cannot give a scope/role combination to another user if you don't have a higher or equivalent additional role.
+
+Scope | Description | Required fields
+---------- | ----------- | -----------
+ORG_BASE | Applies the role on a specific organization | `organization.id`
+ORG_TREE | Applies the role on a specific organization and its sub-organzations | `organization.id`
+ORG_SUBS | Applies the role the sub-organzations of a specific organization | `organization.id`
+ORG_TOPLEVEL | Applies the role on all top-level organizations | &nbsp;
+TAGS_ANYMATCH | Applies the role on all organizations that are tagged with at least one the specified tags | `tags`
+GLOBAL | Applies role on all organizations | &nbsp;
+
+### Get additional roles of user
+
+`GET /users/:id/additional_roles`
+
+```shell
+# Get additional role of user
+curl -X GET "https://cloudmc_endpoint/v2/users/:id/additional_roles" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json"
+
+# Response body example
+```
+```json
+{
+"data": [
+    {
+        "id": "ed18cdd0-0fa0-464d-a993-e689d39dfe2e",
+        "creationDate": "2019-06-06T15:01:22.000Z",
+        "scopeQualifier": "ORG_BASE",
+        "role": {
+            "id": "cc71a40b-d90b-40db-b724-08d71f5179b5",
+            "name": "user",
+            "isSystem": true,
+            "isFixed": true
```

Comment:
Should we rename these to just be `system` and `fixed`?

Reply:
Not really doc related but would make sense.

Reply:
Yeah, not related to this PR, but it struck me when I was reading the doc